### PR TITLE
Fix gRPC interop tests

### LINKED ISSUE #24182: Grpc Interop tests assembly load failure on Helix
The gRPC interop tests are currently disabled.

We saw failures after the latest dependency updates brought breaking Logging changes in the runtime: https://dev.azure.com/dnceng/public/_build/results?buildId=739743&view=results

The error suggests that the wrong version of the M.E.Logging.Console is being loaded at runtime:
```
[ERROR] Unhandled exception. System.TypeLoadException: Could not load type 'Microsoft.Extensions.Logging.Console.SimpleConsoleFormatterOptions' from assembly 'Microsoft.Extensions.Logging.Console, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.
[ERROR]    at InteropTestsWebsite.Program.<>c.<CreateHostBuilder>b__1_0(ILoggingBuilder builder)
[ERROR]    at Microsoft.Extensions.Hosting.HostingHostBuilderExtensions.<>c__DisplayClass5_0.<ConfigureLogging>b__1(ILoggingBuilder builder)
[ERROR]    at Microsoft.Extensions.DependencyInjection.LoggingServiceCollectionExtensions.AddLogging(IServiceCollection services, Action`1 configure)
[ERROR]    at Microsoft.Extensions.Hosting.HostingHostBuilderExtensions.<>c__DisplayClass5_0.<ConfigureLogging>b__0(HostBuilderContext context, IServiceCollection collection)
[ERROR]    at Microsoft.Extensions.Hosting.HostBuilder.CreateServiceProvider()
[ERROR]    at Microsoft.Extensions.Hosting.HostBuilder.Build()
[ERROR]    at InteropTestsWebsite.Program.Main(String[] args) in F:\workspace\_work\1\s\src\Grpc\test\testassets\InteropWebsite\Program.cs:line 33
Writing minidump to file /home/helixbot/dotnetbuild/dumps/dotnet.31827.dmp
Written 93696000 bytes (22875 pages) to core file
Dump successfully written
```

I suspect that the helix environment didn't install the correct aspnetcore runtime and therefore the M.E.Logging.Console binary that was loaded is old.

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes #24182

### CODE REVIEW

**Comment by JamesNK** [comment]
@JunTaoLuo Were these tests just broken, or disabled? Are additional changes required for these tests to run again?

**Comment by JunTaoLuo** [comment]
The tests were disabled because they were failing on Helix after an SDK change, https://github.com/dotnet/aspnetcore/issues/24182. I forgot to ping you on it sorry.

**Comment by JunTaoLuo** [comment]
You need to revert the changes https://github.com/dotnet/aspnetcore/blob/dafe86602a5add7aa134c0c2d695f9dc704cdf05/src/Grpc/test/InteropTests/InteropTests.csproj#L7-L8 to re-enable running the tests.

File: src/Grpc/test/InteropTests/InteropTests.csproj [hunk 1]
```diff
@@ -4,8 +4,7 @@
     <ContainsFunctionalTestAssets>true</ContainsFunctionalTestAssets>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
 
-    <!-- https://github.com/dotnet/aspnetcore/issues/24182 -->
-    <BuildHelixPayload>false</BuildHelixPayload>
+    <BuildHelixPayload>true</BuildHelixPayload>
```
Comment (JunTaoLuo) [inline src/Grpc/test/InteropTests/InteropTests.csproj:7]: You can just remove it, it's default to true.

**Review by JunTaoLuo — APPROVED** [review]

**Pull request merged**